[2022 key rotation docs] Recommend updating the install method (#12559)
DataDog/documentation: content/en/agent/faq/linux-agent-2022-key-rotation.md

## Manual update

> We encourage users to use one of the install methods above which automatically trust the new GPG key, as they will also automatically fetch future keys as well. If this is not an option, these instructions will download and trust the new key.

Reviewer: ```suggestion
Datadog encourages you to use one of the install methods above, which automatically trusts the new GPG key as it also automatically fetches future keys. If this is not an option, the following instructions also downloads and trusts the new key.
```

Author: ```suggestion
Datadog encourages you to use one of the install methods above, which trust the new GPG key as well as all future keys automatically. If this is not an option, the following instructions manually download and trust the new key.
```
